server/container/core/src/main/java/org/apache/james/server/core/MimeMessageCopyOnWriteProxy.java

```diff
@@ -46,6 +47,15 @@
  * automatically cloning it (if shared) when a write operation is invoked.
  */
 public class MimeMessageCopyOnWriteProxy extends MimeMessage implements Disposable {
+    public static class ContentTypeCleaner {
+        public static String cleanContentType(MimePart mimePart, String contentType) {
+            return null;
```

Comment:
is content-type is save to remove?

Reply:
This method is called when Content-Type can not be parsed and when Content-Type is `null`, it is ignored, so I think it pretty safe.

Reply:
We can return a string has been removed all slashes. WDYT?

Reply:
Basically, it's an issue created due to production failure and there are already no slashes.

At this point, we are unable to extract a valid `Content-Type`, making up another one looks dangerous.

@rouazana Maybe I can add a log wan/error, WDYT?

Reply:
@trantienduchn the issue is not having a `/`, it's that there is a `;` while a `/` is expected. The content-type here seems to be really screwed. It should not be "string" but probably "application/pdf". But we have no way to recover it.
Ok for me to add a warn log.